@shared/api/internal.ts

```diff
@@ -1356,13 +1348,6 @@ export const loadSettings = (): Promise<
     type: SERVICE_TYPES.LOAD_SETTINGS,
   });
 
-export const getBlockedDomains = async () => {
-  const resp = await sendMessageToBackground({
-    type: SERVICE_TYPES.GET_BLOCKED_DOMAINS,
-  });
-  return resp;
-};
-
 export const getBlockedAccounts = async () => {
```

Comment:
We'll continue using `blockedAccounts`, but this will be renamed. More info below